resident-default.properties

```diff
@@ -1142,4 +1145,17 @@ resident-transliteration-workaround-for-eng-fra=eng-hin,hin-fra
 resident-transliteration-workaround-for-eng-spa=eng-hin,hin-spa
 
 # Reg-processer-credential-partner-policy-url
-mosip.resident.reg-processer-credential-partner-policy-url=${config.server.file.storage.uri}registration-processor-credential-partners.json
\ No newline at end of file
+mosip.resident.reg-processer-credential-partner-policy-url=${config.server.file.storage.uri}registration-processor-credential-partners.json
+
+hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+## Keymanager service
+#Type of keystore, Supported Types: PKCS11, PKCS12, Offline, JCE
+mosip.kernel.keymanager.hsm.keystore-type=OFFLINE
+# For PKCS11 provide Path of config file.
+# For PKCS12 keystore type provide the p12/pfx file path. P12 file will be created internally so provide only file path & file name.
+# For Offline & JCE property can be left blank, specified value will be ignored.
+mosip.kernel.keymanager.hsm.config-path=/config/softhsm-application.conf
```

Comment:
if used offline, do you still need this property ? Also why we are using keymanager in resident service ?

Reply:
In resident service we are using keygenerator to generate keys 
https://github.com/mosip/resident-services/blob/772b2a4639fc745128828760c998e2f0f424f8e6/resident/resident-service/src/main/java/io/mosip/resident/service/impl/IdAuthServiceImpl.java#L314
So as part of java 21 migration it's not able to create bean  so added https://github.com/mosip/resident-services/blob/772b2a4639fc745128828760c998e2f0f424f8e6/resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java#L47 multiple classes in import so KeyGenerator class needs these configs to be present. so added.